### DISCUSSION TITLE
Broken link on https://github.com/keras-team/keras/blob/master/examples/variational_autoencoder.py

### DISCUSSION THREAD

**Discussion by gpanikian** [keras-team/keras#20472] [General]
Greetings, 
 kindly i would like to inform you that there is a broken link on your site https://blog.keras.io/building-autoencoders-in-keras.html
when viewing the variational autoencoder python file, 
(https://github.com/keras-team/keras/blob/master/examples/variational_autoencoder.py)
Could you please help me to view it?

Best wishes, 
Garo

**Comment by nikolasavic3** [comment]
Hi @Garo,

[This issue might be resolved by now, but in case anyone else finds this:]

The VAE example you're looking for is now hosted at:

https://keras.io/examples/generative/vae/

Let me know if you need anything else!​​​​​​​​​​​​​​​​

Note: The blog post you referenced is from the legacy Keras blog. You can find updated examples and tutorials in the Keras docs at keras.io.​​​​​​​​​​​​​​​​
